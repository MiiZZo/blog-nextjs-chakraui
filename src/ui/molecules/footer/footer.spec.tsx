import React from "react";
import { withRouter } from "next/router";
import { cleanup } from "@testing-library/react";
import { myRender } from "@lib/testing/render";
import { Footer } from "@ui";

afterEach(cleanup);

describe("footer component", () => {
  it("should be match a shapshot", () => {
    const { asFragment } = myRender(<Footer />);

    expect(asFragment()).toMatchSnapshot();
  });
});
