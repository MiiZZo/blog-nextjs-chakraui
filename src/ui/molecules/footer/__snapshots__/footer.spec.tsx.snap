// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`footer component should be match a shapshot 1`] = `
<DocumentFragment>
  <footer
    class="css-70aikg"
  >
    <ul
      class="css-19ndbdu"
    >
      <li
        class="css-992tcj"
      >
        <a
          class="css-u5zpo1"
        >
          Articles
        </a>
      </li>
      <li
        class="css-992tcj"
      >
        <a
          class="css-u5zpo1"
        >
          About
        </a>
      </li>
      <li
        class="css-992tcj"
      >
        <a
          class="css-u5zpo1"
        >
          Help
        </a>
      </li>
    </ul>
  </footer>
</DocumentFragment>
`;
